core/src/translate/query.rs

```diff
@@ -155,29 +191,39 @@ pub fn translate_select_item(sql_select_item: &SqlSelectItem) -> Result<SelectIt
             };
 
             Ok(SelectItem::Expr {
-                expr: translate_expr(expr)?,
+                expr: translate_expr_with_params(expr, params)?,
                 label,
             })
         }
-        SqlSelectItem::ExprWithAlias { expr, alias } => {
-            translate_expr(expr).map(|expr| SelectItem::Expr {
+        SqlSelectItem::ExprWithAlias { expr, alias } => translate_expr_with_params(expr, params)
+            .map(|expr| SelectItem::Expr {
                 expr,
                 label: alias.value.to_owned(),
-            })
-        }
+            }),
         SqlSelectItem::QualifiedWildcard(object_name, _) => Ok(SelectItem::QualifiedWildcard(
             translate_object_name(object_name)?,
         )),
         SqlSelectItem::Wildcard(_) => Ok(SelectItem::Wildcard),
     }
 }
 
-fn translate_table_with_joins(sql_table_with_joins: &SqlTableWithJoins) -> Result<TableWithJoins> {
+pub fn translate_select_item(sql_select_item: &SqlSelectItem) -> Result<SelectItem> {
```

Comment:
It would be great to add a short doc comment here with a `# Errors` section explaining when this Result can fail.